tests/unit/src/main/java/org/mule/tck/junit4/AbstractMuleContextTestCase.java

```diff
@@ -143,6 +144,17 @@ protected void setDisposeContextPerClass(boolean val) {
     disposeContextPerClass = val;
   }
 
+  // If this runs inside a Mule classloader it's automatically loaded, but in unit tests that
+  // are run outside we need to set it up manually.
+  static {
```

Comment:
That would be DefaultMuleContext, where it was before. But it's also loaded from the classloader if running inside a container.